test/functional/p2p_permissions.py

```diff
@@ -119,14 +119,14 @@ def check_tx_relay(self):
             [tx],
             self.nodes[1],
             success=False,
-            reject_reason='{} from peer=0 was not accepted: txn-mempool-conflict'.format(txid)
+            reject_reason='{} (wtxid={}) from peer=0 was not accepted: txn-mempool-conflict'.format(txid, tx.getwtxid())
```

Comment:
nit: we could also change it to use f-strings instead of `format` given it touches this line.

Reply:
Will keep this in mind for the future. I'm just sticking with how this string was formatted previously.